curriculum/challenges/english/25-front-end-development/workshop-registration-form/62b30924c5e4ef0daba23b5e.md

```diff
@@ -31,7 +31,7 @@ Your `fieldset:last-of-type` should have `border-bottom` set as `none`.
 
 ```js
 const borderBottom = new __helpers.CSSHelp(document).getStyle('fieldset:last-of-type')?.borderBottom;
-assert(borderBottom === 'none' || borderBottom === 'medium none' || borderBottom === 'medium');
+assert.include(['none', 'medium none', 'medium'], borderBottom);
```

Comment:
please use `oneOf`, here you are testing `borderBottom` so it should go as first argument, and `oneOf` allows that